src/components/ErrorMessage.tsx

```diff
@@ -15,25 +16,71 @@ const Container = styled.View`
   display: flex;
   align-items: center;
 `
-
-const ErrorText = styled.Text`
+const ErrorTitle = styled.Text`
+  padding: ${props => `${props.theme.spacings.md} ${props.theme.spacings.xl}`} 0 ${props => props.theme.spacings.xl}};
+  font-size: ${props => props.theme.fonts.headingFontSize};
+  font-family: ${props => props.theme.fonts.contentFontBold};
+  color: ${props => props.theme.colors.primary};
+  text-align: center;
+`
+const ErrorText = styled.Text<{ centered?: boolean }>`
   padding: ${props => `${props.theme.spacings.md} ${props.theme.spacings.xl}`};
   font-size: ${props => props.theme.fonts.defaultFontSize};
+  font-family: ${props => props.theme.fonts.contentFontRegular};
+  text-align: ${props => (props.centered ? 'center' : 'left')};
+`
+const NetworkErrorWrapper = styled.View`
+  background-color: ${prop => prop.theme.colors.background};
+  height: 100%;
+  align-items: center;
+`
+const IconStyle = styled.View`
+  width: 33%;
+  height: auto;
+  align-items: center;
+`
+const UpperSection = styled.View`
+  width: 140%;
+  height: 67%;
+  background-color: ${props => props.theme.colors.networkErrorBackground};
+  border-bottom-left-radius: ${hp('60%')}px;
+  border-bottom-right-radius: ${hp('60%')}px;
+  margin-bottom: ${props => props.theme.spacings.xxl};
+  justify-content: center;
+  align-items: center;
+  text-align: center;
 `
```

Comment:
I think we should move this to a separate component that just takes a background color as props since this is (or should be) the same as `UpperSection` in `ExerciseFinishedScreen`.

Reply:
Color and height, yeah. I'll move it to its own component.

Does this need a test...?

Reply:
No, styled components don't need tests. If we were to test css we'd spend 90% of the time with testing :D

Reply:
Why do we need a different height here?

Reply:
Less of a reason of "need", more of a reason of "that's what the Zeplin designs look like" 😄

Reply:
Okay. I'd prefer making it of consistent height if there is not a specific reason why we need another height. The only reason I could imagine why a different height would perhaps make sense is that the status bar/header is of a different color.

Reply:
Just tested it, the usual height for this is 50%, which is just barely enough on my device. Increasing the height is a space issue on exercise-finished screens.

I'm assuming that on smaller devices, it might already overflow. And even if it doesnt, using 50% height leaves a lot of empty space on the screen and just plain doesn't look as good as the current Zeplin design.

Is there a problem with making the height adjustable? I'd just let the component take another optional prop and default to 50%.

Reply:
No problem with adjustable height, especially if it is needed, because of the different content of the screens using it.